### PR TITLE
Fix docstrings with incorrect anchor order

### LINKED ISSUE #73: Anchor is defined in the wrong order
The `Tephigram` class docstring states that the anchor should be defined in [(bottom-left-pressure, bottom-left-temp), (top-right-pressure, top-right-temp)]:

https://github.com/SciTools/tephi/blob/1a3e4b89550fb77dc26674494446d20477f0cadd/tephi/__init__.py#L600-L604

https://github.com/SciTools/tephi/blob/1a3e4b89550fb77dc26674494446d20477f0cadd/tephi/__init__.py#L807-L809

But actually the values are used to specify the bottom-right and top-left corners:

image

### PR DESCRIPTION
**Pull request by ESadek-MO**
#### Closes #73.

- Docstrings updated to reference correct anchor order
- Also added consistency to "temperature - pressure pair" references to always be hyphenated.

### CODE REVIEW

File: tephi/__init__.py [hunk 1]
```diff
@@ -595,9 +595,9 @@ def __init__(
             A :class:`tephi.Locator` instance or a numeric step size
             for the dry adiabat lines.
         * anchor:
-            A sequence of two pressure, temperature pairs specifying the extent
-            of the tephigram plot in terms of the bottom left hand corner and
-            the top right hand corner. Pressure data points must be in units of
+            A sequence of two pressure-temperature pairs specifying the extent
```
Comment (bjlittle) [inline tephi/__init__.py:598]: @ESadek-MO Perhaps the use of `(pressure, temperature) pairs` might be clearer?

File: tephi/__init__.py [hunk 2]
```diff
@@ -839,7 +839,7 @@ def plot(self, data, **kwargs):
 
         Args:
 
-        * data: pressure and temperature pair data points.
+        * data: pressure-temperature pair data points.
```
Comment (bjlittle) [inline tephi/__init__.py:842]: Again `(pressure, temperature)` ?

File: tephi/isopleths.py [hunk 3]
```diff
@@ -140,7 +140,7 @@ def _wet_adiabat_gradient(min_temperature, pressure, temperature, dp):
         calculate the gradient difference.
 
     Returns:
-        The gradient change as a pressure, potential temperature value pair.
+        The gradient change as a pressure-potential_temperature value pair.
```
Comment (bjlittle) [inline tephi/isopleths.py:143]: @ESadek-MO Here `(pressure, potential-temperature)` is definitely better

**Review by bjlittle — CHANGES_REQUESTED** [review]
@ESadek-MO Only some minor suggestions to service, otherwise lookin' good!

File: tephi/isopleths.py (hunk 3 above)
Comment (ESadek-MO) [reply tephi/isopleths.py]: You're definately right in all cases, have updated to reflect.

**Review by bjlittle — APPROVED** [review]

**Pull request merged**